README.md

```diff
@@ -3297,7 +3330,7 @@ Other Style Guides
   <a name="naming--leading-underscore"></a><a name="22.4"></a>
   - [23.4](#naming--leading-underscore) Do not use trailing or leading underscores. eslint: [`no-underscore-dangle`](https://eslint.org/docs/rules/no-underscore-dangle.html)
 
-    > Why? JavaScript does not have the concept of privacy in terms of properties or methods. Although a leading underscore is a common convention to mean “private”, in fact, these properties are fully public, and as such, are part of your public API contract. This convention might lead developers to wrongly think that a change won’t count as breaking, or that tests aren’t needed. tl;dr: if you want something to be “private”, it must not be observably present.
+    > Why? ECMA2022 introduced [`#` as the private identifier](https://tc39.es/ecma262/#sec-static-semantics-privateboundidentifiers). Although a leading underscore is a common convention to mean “private”, in fact, these properties are fully public, and as such, are part of your public API contract. This convention might lead developers to wrongly think that a change won’t count as breaking, or that tests aren’t needed. tl;dr: if you want something to be “private”, it must not be observably present.
```

Comment:
technically, `#` is part of the name - it's not a "private identifier". `#foo` isn't "private foo", it's a distinct name from `foo`.

Reply:
Hey @ljharb, thank you so much for your comment! [`#`](https://tc39.es/ecma262/#prod-PrivateIdentifier) is defined as an official reserved keyword in JavaScript syntax. With `#` prefix, [we can create private members](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields).

```javascript
class ClassWithPrivateField {
  #privateField;

  constructor() {
    this.#privateField = 42;
    delete this.#privateField;   // Syntax error
    this.#undeclaredField = 444; // Syntax error
  }
}

const instance = new ClassWithPrivateField()
instance.#privateField === 42;   // Syntax error
```

Reply:
No, it very much is not a "reserved keyword". What you're linking to is a grammar production in the specification, which has nothing to do with reserved words or keywords.

The MDN article you linked says what I did:
> The # is a part of the name itself.

I was an editor of the specification when the class fields proposal was merged; I'm quite certain of what I'm talking about.

Reply:
Understood:) Thanks a lot @ljharb! I learned a lot from your comments.

Reply:
```suggestion
    > Why? ECMA2022 introduced [**# names**](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) to create private class members with a `#` prefix. Although a leading underscore is a common convention to mean “private”, in fact, these properties are fully public, and as such, are part of your public API contract. This convention might lead developers to wrongly think that a change won’t count as breaking, or that tests aren’t needed. tl;dr: if you want something to be “private”, it must not be observably present.
```